xbmc/settings/MediaSourceSettings.h

```diff
@@ -36,7 +36,7 @@ class CMediaSourceSettings : public ISettingsHandler
   bool Save(const std::string &file) const;
   void Clear();
 
-  VECSOURCES* GetSources(const std::string &type);
+  std::vector<CMediaSource>* GetSources(const std::string& type);
```

Comment:
It's odd that this doesn't return a reference, it looks like whenever this function is used there is no null pointer check.